src/darker/config.py

```diff
@@ -97,6 +99,21 @@ def validate_config_output_mode(config: DarkerConfig) -> None:
     )
 
 
+def override_color_with_environment(pyproject_config: DarkerConfig) -> DarkerConfig:
+    """Override ``color`` if the ``PY_COLORS`` environment variable is '0' or '1'
+
+    :param config: The configuration read from ``pyproject.toml``
+    :return: The modified configuration
+
+    """
+    py_colors = os.getenv("PY_COLORS")
+    if py_colors not in {"0", "1"}:
```

Comment:
I think `true` or `false` should also be accepted. (Including all capital case variants)